main/src/cgeo/geocaching/enumerations/CacheType.java

```diff
@@ -36,6 +36,7 @@
     GPS_EXHIBIT("gps", "GPS Adventures Exhibit", "72e69af2-7986-4990-afd9-bc16cbbb4ce3", R.string.gps, R.drawable.type_event, "1304", R.drawable.dot_event), // icon missing
     BLOCK_PARTY("block", "Geocaching HQ Block Party", "bc2f3df2-1aab-4601-b2ff-b5091f6c02e3", R.string.block, R.drawable.type_event, "4738", R.drawable.dot_event), // icon missing
     LOCATIONLESS("locationless", "Locationless (Reverse) Cache", "8f6dd7bc-ff39-4997-bd2e-225a0d2adf9d", R.string.locationless, R.drawable.type_virtual, "12", R.drawable.dot_virtual), // icon missing
+    ADVLAB("advlab", "Adventure Lab Cache", "", R.string.advlab, R.drawable.type_advlab, "7777", R.drawable.dot_advlab),
```

Comment:
Official GC Cache types are defined in https://www.geocaching.com/about/cache_types.aspx
Therefore I would stay with `Lab Cache`. 
The `pattern` and `wptTypeId` are used/defined in the cache website. But we have no cache site for lab caches. Is the `wptTypeId` defined/used in the lab cache data? 

This additional row created the CI error 
`cgeo.geocaching.apps.AbstractLocusAppTest > testToLocusTypeCount FAILED`
Please increase the count number in the test.
The Locus app does (currently) not support lab caches, see 
https://github.com/asamm/locus-api/blob/82d0b51472dc71420df0fea23280cb34dc97ee8c/locus-api-core/src/main/java/locus/api/objects/geocaching/GeocachingData.kt#L623

Reply:
Adventure Lab Cache -> Lab Cache: fixed

cgeo.geocaching.apps.AbstractLocusAppTest > testToLocusTypeCount FAILED .... increased from 10 to 11, not knowing the logic behind all this at this time but will follow up later.

With respect to `pattern` and `wptTypeId`: No, none of those come down via the API method. Now: what does that mean?

Reply:
> cgeo.geocaching.apps.AbstractLocusAppTest > testToLocusTypeCount FAILED .... increased from 10 to 11, not knowing the logic behind all this at this time but will follow up later.

Fine.
The logic here is to detect that we changed something at the caches types, so it is necessary to check if we need to configure the locus connector. This is ok for now.

> With respect to `pattern` and `wptTypeId`: No, none of those come down via the API method. Now: what does that mean?

Let them blank.